docs/content/cache.md

```diff
@@ -364,11 +364,11 @@ will need to be cleared or unexpected EOF errors will occur.
 - Default:     5M
 - Examples:
     - "1m"
```

Comment:
minor: should probably be `1M`

Reply:
Probably.. Although it is not *wrong*: "1m", "1M", "1Mi", "1MI", "1mi", "1MiB", "1mib", "1mIB"...etc.. all are accepted and with identical results. All other examples use the "rclone standard" single letter uppercase, though, so I will update this also.
Thank you for reporting, really great that you are willing to have a look!

Reply:
I noticed you corrected it in `backend/cache/cache.go` line `101` so I thought it was worth pointing out, even if very minor.
By the way thank _you_ for your amazing work!

Reply:
> I noticed you corrected it in backend/cache/cache.go line 101 so I thought it was worth pointing out, even if very minor.

Oh, right, I agree!